### ISSUE TITLE
"Copy code ... from the Pantheon Dev environment" can be confusing when working with multidev

### ISSUE THREAD

**Issue by joe-westcott** [pantheon-systems/localdev-issues#75]
In Localdev, the option to pull code shows this text:

> Copy code (your website’s CSS/JS/HTML/PHP) from the Pantheon Dev environment to your local.

This text is accurate when pulling from the Dev environment -- note the capital letter "D" -- but this can lead to confusion when the corresponding "Cloud Site" is actually a multidev environment, not Dev. I think this text might just be an artifact from when Localdev could only pull from Dev, and not multidev envs.

Suggested fix:

Update the text to something like the following:

> Copy code (your website’s CSS/JS/HTML/PHP) from the matching Pantheon environment (Dev or multidev) to your local.

You might also update this reference to "Dev" in the other Push and Pull options in Localdev.

For example, the "Pull database" option and the "Push files" option indicate that they'll push or pull from the "Dev" environment.

Thank you for considering it!

Joe

**Comment by joe-westcott** [comment]
Here's a screenshot of the text in question, if it helps.

![localdev-dev-env-text-mention-even-on-multidev](https://user-images.githubusercontent.com/13838392/87205507-99a87c80-c2d5-11ea-9e78-04df5ac91bb3.png)

**Comment by pirog** [comment]
@joe-westcott good call. We should def put the multidev env there if you're on a multidev envvar.

@dustinleblanc might be good to add this to your UX sprint
